libs/admin/src/lib/components/users/users.component.html

```diff
@@ -1,39 +1,49 @@
-<div>
-  @if (selectedUser) {
-  <asset-sg-user-edit [user]="selectedUser" [workgroups]="workgroups"></asset-sg-user-edit>
-  }
-
-  <table mat-table [dataSource]="users" class="mat-elevation-z8 table">
+<div class="users">
+  <div class="users__header">
+    <h1 translate>admin.users</h1>
+  </div>
+  <table mat-table [dataSource]="users" class="mat-elevation-z8 users__table">
     <ng-container matColumnDef="email">
-      <th mat-header-cell *matHeaderCellDef>Name</th>
+      <th mat-header-cell *matHeaderCellDef translate>admin.email</th>
       <td mat-cell *matCellDef="let user">{{ user.email }}</td>
     </ng-container>
-    <ng-container matColumnDef="workgroups">
-      <th mat-header-cell *matHeaderCellDef>Workgroups</th>
-      <td mat-cell *matCellDef="let user">
-        @for (workgroup of user.workgroups; track workgroup.workgroupId) {
-        <div>{{ workgroup.workgroup.name }}: {{ workgroup.role }}</div>
-        }
-      </td>
-    </ng-container>
     <ng-container matColumnDef="isAdmin">
-      <th mat-header-cell *matHeaderCellDef>Admin</th>
+      <th mat-header-cell *matHeaderCellDef translate>admin.userPage.admin</th>
 
       <td mat-cell *matCellDef="let user">
-        <mat-checkbox [checked]="user.isAdmin" (change)="update(user, $event)"></mat-checkbox>
+        <mat-checkbox [checked]="user.isAdmin" (change)="updateIsAdminStatus(user, $event)"></mat-checkbox>
       </td>
     </ng-container>
     <ng-container matColumnDef="languages">
-      <th mat-header-cell *matHeaderCellDef>Sprache</th>
+      <th mat-header-cell *matHeaderCellDef translate>admin.userPage.lang</th>
       <td mat-cell *matCellDef="let user">{{ user.lang }}</td>
     </ng-container>
+    <ng-container matColumnDef="workgroups">
+      <th mat-header-cell *matHeaderCellDef translate>admin.workgroups</th>
+      <td
+        mat-cell
+        *matCellDef="let user"
+        [matTooltip]="formatWorkgroupsTooltip(user.workgroups)"
+        matTooltipClass="workgroups-tooltip"
+      >
```

Comment:
Position the tooltip left so it appears right next to the text.
```suggestion
      <td
        mat-cell
        *matCellDef="let user"
        [matTooltip]="formatWorkgroupsTooltip(user.workgroups)"
        matTooltipPosition="left"
        matTooltipClass="workgroups-tooltip"
      >
```